CHANGELOG.md

```diff
@@ -3,6 +3,60 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.0.0-beta.2
+
+_Sep 23, 2021_
+
+A big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🇻🇳 Add Vietnamese (viVN) locale (#2668) @tuananh281098
+- 🇵🇱 Improve Polish (plPL) locale (#2632) @michallukowski
+- ⚡️ Apply the `valueFormatter` to `singleSelect` select options (#2581) @DanailH
+
+### `@mui/x-data-grid@v5.0.0-beta.2` / `@mui/x-data-grid-pro@v5.0.0-beta.2`
+
+#### Breaking changes
+
+- [DataGrid] The params passed to the `valueFormatter` were changed. (#2581) @DanailH
+  Use the `api` to get the missing params.
```

Comment:
To fix the formatting: https://github.com/mui-org/material-ui-x/blob/65a0671b2983a63d6107cebf7e0cb02093a05ed9/CHANGELOG.md

```suggestion

  Use the `api` to get the missing params.
```